.github/workflows/test-jobs.yml

```diff
@@ -39,20 +39,13 @@ jobs:
         name: Test
       - name: Test Results
         if: always()
-        uses: mikepenz/action-junit-report@v2
+        uses: mikepenz/action-junit-report@v3
         with:
           report_paths: 'build/output/junit-*.xml'
           github_token: ${{ secrets.GITHUB_TOKEN }}
           fail_on_failure: true
           require_tests: true
           check_name: Unit Test Results
-        # This is the only way to get test reports out from GitHub Actions
-        # TODO remove this before release
-      - uses: actions/upload-artifact@v2
```

Comment:
Still could be useful to investigate failures. I hope you never have one though.

Reply:
I've re-added these, just changed the condition to `failed()` rather than `always()`